constellations/gemini-admin/src/home/homeRegistrations.test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import { HomeTabSectionRegistrations } from "./homeRegistrations.js";
+
+describe("Home Registrations", () => {
+    const component = shallow(<HomeTabSectionRegistrations />);
+
+    test("renders", () => {
+        expect(component.exists()).toBe(true);
+        expect(component.find("h3").text()).toContain(
+            "Pending Registrations For Classes"
+        );
+        expect(component.find("Link").text()).toContain("View By Class");
+        expect(component.find("h3").text()).toContain(
+            "New Registrations For AFH"
+        );
+        expect(component.find("Link").text()).toContain("View By AFH Session");
```

Comment:
In this component looks like there are multiple h3s and Links. If this is the case, you'll have to use the `.at(...)`
So just do something like this: `component.find("h3").at(0).text()` which will find the first h3 in the component. Of course, `.at(1)` will find the second!